include/PandoraMonitoring.h

```diff
@@ -406,12 +434,33 @@ class PandoraMonitoring
     const pandora::Pandora *const   m_pPandora;                 ///< The associated pandora instance
     TApplication                   *m_pApplication;             ///< The root application
     TEveManager                    *m_pEveManager;              ///< The eve manager
+    TEveManager                    *m_pEveManager2D;            ///< The eve manager for 2D hits
     TTreeWrapper                   *m_pTreeWrapper;             ///< wrapper around TTree functionality
 
+    TEveViewer                     *m_p3DView;                  ///< Viewer for the 3D view
+    TEveViewer                     *m_p2DUView;                 ///< Viewer for the U 2D view
+    TEveViewer                     *m_p2DVView;                 ///< Viewer for the V 2D view
+    TEveViewer                     *m_p2DWView;                 ///< Viewer for the W 2D view
+
+    TEveScene                      *m_p3DEventScene;            ///< Scene containing the 3D event
+    TEveScene                      *m_p2DUEventScene;           ///< Scene containing the U 2D event
+    TEveScene                      *m_p2DVEventScene;           ///< Scene containing the V 2D event
+    TEveScene                      *m_p2DWEventScene;           ///< Scene containing the W 2D event
+
+    TEveScene                      *m_p3DGeometryScene;         ///< Scene containing the 3D geometry
+    TEveScene                      *m_p2DUGeometryScene;        ///< Scene containing the U 2D geometry
+    TEveScene                      *m_p2DVGeometryScene;        ///< Scene containing the V 2D geometry
+    TEveScene                      *m_p2DWGeometryScene;        ///< Scene containing the W 2D geometry
+
```

Comment:
Need to ensure these are all initialised to e.g. nullptr in constructors to ensure in well-defined state.

Could replace existing NULL use case with nullptr as required, too?

Reply:
In c++11 one can do "default member initializer"
http://en.cppreference.com/w/cpp/language/data_members#Member_initialization
e.g.,
```c++
TEveScene                      *m_p2DWGeometryScene=nullptr;
```
Which is then the default value, unless explicitly set to something else via constructor initializer list.

Reply:
Will change all NULL to nullptr.